server.js

```diff
@@ -437,6 +449,30 @@ app.post('/submitanswer', function(req, res) {
     });
 });
 
+/* check if the submitted MC answer is correct */
+app.post('/submitMCanswer', function(req, res) {
```

Comment:
No need for a new route. Wait for my changes and you can use the same route

Reply:
We do need new route for every question, but okay will wait for what you have done